docs/loopback/datasources.md

```diff
@@ -0,0 +1,72 @@
+# Datasources
+
+## Remote connectors
+
+Loopback provides connectors which enable your application to consume data from remote services
+using different protocols (such as `loopback-connector-remote`, `loopback-connector-rest` and
+`loopback-connector-soap`).
+
+### Remote vs. Rest
+
+At first sight the remote connector simply seems to be a wrapper for the underlying protocol (REST
+by default). But during the evaluation of the connectors, it seemed to be the case, that the remote
+connector has some important characteristics:
+
```

Comment:
From the [README](https://github.com/strongloop/loopback-connector-remote) of the loopback-connector-remote

> The remote connector enables you to use a LoopBack application as a data source via REST. You can use the remote connector with a LoopBack application, a Node application, or a browser-based application that uses LoopBack in the client. The connector uses Strong Remoting.
> 
> In general, using the remote connector is more convenient than calling into REST API, and enables you to switch the transport later if you need to.

So may we want to note that it's specifically designed to use loopback apps as a remote datasource

Reply:
This is actually what confuses me. The docs seem to imply that it is only a wrapper which allows you to exchange the transport.